docs/one-wire-devices.md

```diff
@@ -0,0 +1,150 @@
+</pre>
+
+**Step 2:** Perform a check to find if slave devices are connected or not. 
+
+<pre>
+<code>cat /sys/devices/w1_bus_master1/w1_master_slave_count</code>
+</pre>
+
+**Step 3:** The command in step 2 will return the count of the number of slave devices connected. 
+
+| Result            | Interpretation                            |
+|-------------------|-------------------------------------------|
+| 1                 | A slave device is connected               |
+| 0                 | A slave device is connected               |
+
+**NOTE:** The One-Wire bus master kernel module scans the data pin every 10 seconds for new devices connected, if any, hence, it is recommended to wait a little while and try again. 
+
+**Step 4:** If a slave device is connected, run the following command: 
+
+<pre>
+<code>cat/sys/devices/w1_bus_master1/w1_master_slaves </code>
+</pre>
+
+This will return a (newline delimited) list of your slave device ID(s). The Device ID is the serial number of your One-Wire device. Example of a device ID: 28-000123456789. 
+
+**NOTE:** Each device will have a different serial number, making it complex to use One-Wire devices programmatically. 
+
+<h2 id="read">Read Data from a Connected One-Wire Device</h2>
+
+**Step 1:** Navigate to the specific slave device directory and access the data from the __w1_slave file__ by running the following command: 
+
+<pre>
+<code>cat /sys/devices/w1_bus_master1/DeviceID/w1_slave</code>
+</pre>
+
+The `<DeviceID>` in the aforesaid command is the serial number of your One-Wire device. Please refer to **Step 4** of the aforesaid section "Find Connected One-Wire Devices" to learn more about finding the "DeviceID". 
+
+Assuming the slave device is a temperature sensor "DS18B20", having a serial number- 28-000123456789, the command will be: 
+
+<pre>
+<code>cat /sys/devices/w1_bus_master1/28-000123456789/w1_slave</code>
+</pre>
+
+**Step 2:** The command executed in **Step 1** will return the raw data from the connected One-Wire device. 
+
+Example of the raw data from the connected temperature sensor like the DS18B20: 
+
+<pre>
+<code>b1 01 4b 46 7f ff 0c 10 d8 : crc=d8 YES
+b1 01 4b 46 7f ff 0c 10 d8 t=27062</code>
+</pre>
+
+In the sample raw data above, the final `t=27062` indicates the temperature is 27.062 ˚C.
+
+<h2 id="common">Common Pitfalls and Precautions</h2>
+
+☑️ Cross-check your wiring to make sure that VCC, GND, and DQ lines are correctly connected because incorrect wiring might lead to communication issues.  
+
+☑️ Please ensure that the One-wire device’s dataline is connected to GPIO19. 
+
+☑️ While connecting the One-Wire device to the Omega2 board, refer to the datasheet and documentation for your specific One-Wire device for detailed wiring and configuration instructions.
+
+☑️ Some One-Wire devices will require a pull-up resistor on the Data line (DQ). For example, the popular DS18B20 temperature sensor requires a 4.7 kΩ pull-up resistor on the DQ to operate properly
```

Comment:
Please use the markdown list syntax: https://www.markdownguide.org/basic-syntax/#unordered-lists
It will be more legible in the final output